katalog/thanos/base/components/alert-rules/absent-rules.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: thanos-component-absent
+  namespace: "monitoring"
+  labels:
+    app.kubernetes.io/name: thanos
+    app.kubernetes.io/version: v0.30.2
+spec:
+  groups:
+  - name: thanos-component-absent
+    rules:
+    - alert: ThanosCompactIsDown
+      annotations:
+        description: ThanosCompact has disappeared. Prometheus target for the component cannot be discovered.
+        runbook_url: https://github.com/thanos-io/thanos/tree/main/mixin/runbook.md#alert-name-thanoscompactisdown
+        summary: Thanos component has disappeared.
+      expr: |
+        absent(up{job=~".*thanos-compact.*"} == 1)
+      for: 5m
+      labels:
+        severity: critical
+    - alert: ThanosQueryIsDown
+      annotations:
+        description: ThanosQuery has disappeared. Prometheus target for the component cannot be discovered.
+        runbook_url: https://github.com/thanos-io/thanos/tree/main/mixin/runbook.md#alert-name-thanosqueryisdown
+        summary: Thanos component has disappeared.
+      expr: |
+        absent(up{job=~".*thanos-query.*"} == 1)
+      for: 5m
+      labels:
+        severity: critical
+    - alert: ThanosReceiveIsDown
+      annotations:
+        description: ThanosReceive has disappeared. Prometheus target for the component cannot be discovered.
+        runbook_url: https://github.com/thanos-io/thanos/tree/main/mixin/runbook.md#alert-name-thanosreceiveisdown
+        summary: Thanos component has disappeared.
+      expr: |
+        absent(up{job=~".*thanos-receive.*"} == 1)
+      for: 5m
+      labels:
+        severity: critical
```

Comment:
AFAIK we are not deploying Thanos Receiver, so this alert would be constantly triggering.

We should not ship this alert if we are not deploying the Receiver component.

Reply:
it's right, i remove this from the ruler configuration as well

Reply:
done